### DISCUSSION TITLE
Rename repo name from "kafka-ui" to "kafbat-ui"

### DISCUSSION THREAD

**Discussion by hongbo-miao** [kafbat/kafka-ui#478] [Tech Proposals]
Maybe it is worthing to rename from "kafka-ui" to "kafbat-ui" at some point ☺️

**Comment by Haarolean** [comment]
We decided to keep the name (for now) for a seamless migration and to be able to catch some traffic from the old repo/project

**Reply by hongbo-miao** [reply]
Yeah, that is what I thought so, makes sense. In future worth renaming. Initially I got confused for these two projects.
